README.md

```diff
@@ -239,4 +239,34 @@ or unmount the space when you log out of Data Lab:
 ```
 datalab logout --unmount=/tmp/vospace
 ```
--->
\ No newline at end of file
+-->
+
+## Docker Tests
+
+For convenience, you can build a Docker image to run and test the Data Lab code in an isolated environment. This is particularly useful for ensuring consistent test results across different systems or for users who prefer to run the application in Docker.
```

Comment:
Is docker required to run the tests? We could maybe document what the testing would look like without docker (although I will be one of the Docker users :smile_cat: ). I think this suggestion wouldn't be related to what you added though .. it is just documentation we should have had already I suppose. Maybe a new issue for that?

Reply:
Mark the entire section "Docker Tests" clearly as "totally optional". Users reading the README file should be absolutely clear that that don't have to consider the section at all if they don't want to. (always think of users as interested in running a tool to get data, not to do testing of any kind).

Reply:
The docker thing was more of a tool towards a possible CI workflow but also, or so I thought, maybe something useful to the users? If it creates more confusion than help I'll take it away.

Reply:
Or consider a separate .md file for this, maybe TESTING.md (or something that's quasi-standard out there).

Reply:
If we are going to have test files and docker files in this repository then we should document them in some form. I think creating another markdown file as a "Development Guide" would make sense so we can separate some of these usage and development details. This would also address Robert's concern. If we go this route we then we should include something like in my original comment (as new copy in that file or as a new issue :ok_hand: )

Whether or not we should have these files in here is another question. My thought is that it doesn't seem to hurt anything having these docker and tests files there and might make it easier for others to contribute and for us to develop workflows (as you pointed out).

Reply:
Agree with @chaduhduh but I feel we need to think about it a bit more first. @iglesu do you mind removing this from this branch/PR, and open an issue, e.g. labeled "Add developer info", where we flesh this out (which would be public b/c of the repo being public), or maybe better, a Jira issue?